aten/src/ATen/native/cuda/Math.cuh

```diff
@@ -1265,6 +1265,282 @@ const auto erfcx_string = jiterator_stringify(
+                +3.76329711269987889006e-20,
+        };
+
+        static const T FD[] = {
+                +7.51586398353378947175e-01,
+                +1.16888925859191382142e-01,
+                +6.44051526508858611005e-03,
+                +1.55934409164153020873e-04,
+                +1.84627567348930545870e-06,
+                +1.12699224763999035261e-08,
+                +3.60140029589371370404e-11,
+                +5.88754533621578410010e-14,
+                +4.52001434074129701496e-17,
+                +1.25443237090011264384e-20,
+        };
+
+        static const T GN[] = {
+                +5.04442073643383265887e-01,
+                +1.97102833525523411709e-01,
+                +1.87648584092575249293e-02,
+                +6.84079380915393090172e-04,
+                +1.15138826111884280931e-05,
+                +9.82852443688422223854e-08,
+                +4.45344415861750144738e-10,
+                +1.08268041139020870318e-12,
+                +1.37555460633261799868e-15,
+                +8.36354435630677421531e-19,
+                +1.86958710162783235106e-22,
+        };
+
+        static const T GD[] = {
+                +1.47495759925128324529e+00,
+                +3.37748989120019970451e-01,
+                +2.53603741420338795122e-02,
+                +8.14679107184306179049e-04,
+                +1.27545075667729118702e-05,
+                +1.04314589657571990585e-07,
+                +4.60680728146520428211e-10,
+                +1.10273215066240270757e-12,
+                +1.38796531259578871258e-15,
+                +8.39158816283118707363e-19,
+                +1.86958710162783236342e-22,
+        };
+
+        constexpr T PI = 3.141592653589793238462643383279502884;
+
+        if (std::isinf(x)) {
+            T c = 0.5;
+
+            if (x < 0.0) c = -c;
+
+            return c;
+        }
+
+        T abs_x = std::fabs(x);
+
+        T abs_x_squared = abs_x * abs_x;
+
+        if (abs_x_squared < T(2.5625)) {
+            T t = abs_x_squared * abs_x_squared;
+
+            T cn = 0;
+            T cd = 0;
+
+            for (size_t index = 0; index <= 5; index++) cn = cn * t + CN[index];
+            for (size_t index = 0; index <= 6; index++) cd = cd * t + CD[index];
+
+            T c = abs_x * cn / cd;
+
+            if (x < 0.0) c = -c;
+
+            return c;
+        }
+
+        if (abs_x > T(36974.0)) {
+            T c = T(0.5) + T(1.0) / (PI * abs_x) * std::sin(PI * abs_x * abs_x / T(2.0));
+
+            if (x < 0.0) c = -c;
+
+            return c;
+        }
+
+        T t = PI * (abs_x * abs_x);
+
+        T u = T(1.0) / (t * t);
+
+        T fn = 0;
+        T fd = 0;
+
+        for (size_t index = 0; index <= 9; index++) fn = fn * u + FN[index];
+        for (size_t index = 0; index <= 9; index++) fd = fd * u + FD[index];
+
+        T gn = 0;
+        T gd = 0;
+
+        for (size_t index = 0; index <= 10; index++) gn = gn * u + GN[index];
```

Comment:
nit: (a nit is a note that does not need to be addressed because it's too small, or is just a reviewer's pet peeve, this particularly nit is more the latter) 

type `index` like the type of the condition (in this case, int). This is usually nbd but in this case it's actually an unsigned vs signed comparison (because size_it is an unsigned type https://en.cppreference.com/w/cpp/types/size_t). What I believe happens here is that 10 will be converted to an unsigned type for the comparison, but I find it nicer not to have to reason about these concerns.

Reply:
Will do! And, again, I want to emphasize that I really appreciate nits while on-boarding onto this repository! 🙏